app/views/service_worker/index.js.erb

```diff
@@ -137,6 +137,22 @@
       return;
     }
 
+    // Fetch and/or JS and CSS assets for better persistence than memory cache.
+    if (url.pathname.startsWith("/assets/") || url.pathname.startsWith("/packs/")) {
```

Comment:
This makes sense and will fix the issue. I recommend we do a test drive of this fix with @jdoss   to ensure the fix is solid or did you do that already @benhalpern?